app/src/main/java/com/dozingcatsoftware/vectorpinball/util/IOUtils.java

```diff
@@ -1,8 +1,11 @@
 package com.dozingcatsoftware.vectorpinball.util;
 
+import android.os.Build;
```

Comment:
I'm trying to limit Android dependencies to the "bouncy" package, so I'd rather keep this file as is.